thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionTaskRunner.java

```diff
@@ -86,7 +86,7 @@ private void exploreDimensionsAndAnalyze(TimeSeriesResponse finalResponse) {
     Map<DimensionKey, MetricTimeSeries> res =
         timeSeriesResponseConverter.toMap(finalResponse, collectionDimensions);
     for (Map.Entry<DimensionKey, MetricTimeSeries> entry : res.entrySet()) {
-      if (entry.getValue().getTimeWindowSet().size() < 2) {
+      if (entry.getValue().getTimeWindowSet().size() < 1) {
```

Comment:
Earlier this map used to get baseline and current, which would get separated later on. This was refactored to add a utility which is doing it before hand. So now we get just 1 timeseries in the response.